### ISSUE TITLE
Show where a template/partial was included

### ISSUE THREAD

**Issue by hinrik** [duncanbeevers/rails_view_annotator#3]
Could the begin/end comments show which file (and line) included the partial/template in question? Then you would know exacty which code is responsible for it. Something like this:

``` html
<!-- begin: app/views/some_view/_partial.html.erb (from app/views/different_view/foobar.html.erb:78)-->
<div>
  foobar
</div>
<!-- end: app/views/some_view/_partial.erb (from app/views/different_view/foobar.html.erb:78) -->
```

**Comment by duncanbeevers** [comment]
Hmm, I threw this together in the simplest possible way, but it's still a monkey-patch. I really only use the information already passed in to PartialRenderer#render. It might be possible to figure out the parent view by examining the call stack, but other than that I think we'd have to pass more information into the render method.

**Comment by duncanbeevers** [comment]
I pushed an experimental version of this functionality here: c47d8fc1438e2e6cb6030a59bb58690f8ce77006

I'd love if you could let me know if this does what you want, and it would be especially helpful to get some benchmarks of how this affects and app in development mode. My fear is that it affects performance pretty drastically but I'd love to be wrong.

**Comment by duncanbeevers** [comment]
Surprisingly, my informal benchmark comparing this change to the published version shows no appreciable difference.

**Comment by hinrik** [comment]
I didn't feel a noticable difference either on a reasonably-sized app (https://github.com/rbjarnason/open-active-democracy).

**Comment by duncanbeevers** [comment]
Published at 0.0.6.
